css/styles.css

```diff
@@ -0,0 +1,9 @@
+body {
+  background-color: rgb(134, 42, 110);
+}
+h1 {
+  color: rgb(226, 43, 110);
+  font-family: "Courier New", Courier, monospace;
+  display: flex;
+  justify-content: center;
```

Comment:
You don't have to use a flexbox here, since you have no children elements (just text content). I think `text-align: center` should work as well, or?

Reply:
yes, you are right. I change this for the moment. 
I thought I will add a box around h1 and use this already. ^^
thank you very much =)